scraper/src/github.py

```diff
@@ -71,6 +71,14 @@ def append(self, user, event):
                 "authored_issue_and_pr": [],
             }
 
+    def parse_linked_issues(self, pr_body):
+        if isinstance(pr_body, str):
+            pattern = r"#(\d+)"
+            matches = re.findall(pattern, pr_body)
```

Comment:
1. Issues can be linked to a PR even without having to mention the issue number in the PR body. Those may not be captured this way.

2. We may reference an issue in the PR body, but it may not necessarily solve the issue. Use actual.


Refer all the ways in which link an issue to the PR here: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue